src/app/services/SocialAuth/socialauth.service.spec.ts

```diff
@@ -1,15 +1,15 @@
-import { TestBed } from '@angular/core/testing';
-import { LoginService } from './socialauth.service';
+import { TestBed } from "@angular/core/testing";
+import { LoginService } from "./socialauth.service";
 
 import {
   HttpClientTestingModule,
   HttpTestingController
-} from '@angular/common/http/testing';
-import { environment } from 'src/environments/environment';
-import { of, Observable } from 'rxjs';
-import { APPCONFIG } from 'src/app/config';
+} from "@angular/common/http/testing";
+import { environment } from "src/environments/environment";
+import { of, Observable } from "rxjs";
+import { APPCONFIG } from "src/app/config";
```

Comment:
@patrickf949, let us move everything in the config file into the environments file, delete the file and refactor all usages accordingly.

Reply:
Thanks @CryceTruly done